python/cuml/svm/svc.pyx

```diff
@@ -214,7 +216,7 @@ class SVC(SVMBase, ClassifierMixin):
     coef_ : float, shape (1, n_cols)
         Only available for linear kernels. It is the normal of the
         hyperplane.
-        coef_ = sum_k=1..n_support dual_coef_[k] * support_vectors[k,:]
+        ``coef_ = sum_k=1..n_support dual_coef_[k] * support_vectors[k,:]``
```

Comment:
ah, so it's double backticks here too -- should every code block in a comment be doubled like that?

Reply:
So there were two issues here. If you use a trailing underscore in reST, that means you want to link to an object. So above, `coef_` means link to the object named `coef`. Since no label has been created with the name `coef`, it generated the following error:
```
WARNING: Unknown target name: "coef"
```
Since this section of text was in fact inline code, surrounding it in double backticks correctly displays it as code and also removes the link. So two birds, one stone.